lib/jgi_mg_assembly/runner/pipeline.py

```diff
@@ -56,7 +55,6 @@ def run(self, params):
         report_info = self._build_and_upload_report(pipeline_output,
```

Comment:
I'm not able to comment on the right line (line 41/42), but a shorter way to do the same thing is bool(params.get('skip_rqcfilter'))

Reply:
Good idea. Fixed.